### PR TITLE
#547: RHEL 6 RPM Install Script Infrastructure

### PR DESCRIPTION
**Pull request by tst-ppenev**
* Merged @tst-eclamar's Fusion install scripts.

* Converted `gevars.sh` into a Groovy template expanding the Open GEE version.

* Added a key-value store that can be used to communicate values from one install script to another.

* Added an `installUtils` template that includes the expanded `gevars.sh`, and the key-value store library.

* Rewrote some `opengee-common` and `opengee-fusion` install scripts on the new infrastructure.

### CODE REVIEW

File: earth_enterprise/rpms/build.gradle [hunk 1]
```diff
@@ -239,32 +286,60 @@ task openGeeServerDeb (type: Deb) {
...
+    version = openGeeVersion
+    user = 'root'
+    permissionGroup = 'root'
+    packageGroup = 'Application/Productivity'
+    summary = 'Open GEE Fusion UI and Command-line Utilities'
+    packageDescription =
+'''Command line utitiles and Qt UI for building Open GEE globes.
+
+The utilities in this package allow Open GEE globe and map databases to be
+built from raster, vector and location properties data.
```
Comment (tst-lsavoie) [inline earth_enterprise/rpms/build.gradle]: Minor: I like having oxford commas.

File: earth_enterprise/rpms/shared/lib/key-value.sh [hunk 2]
```diff
@@ -0,0 +1,219 @@
+# Copyright 2017 the Open GEE Contributors.
```
Comment (tst-lsavoie) [inline earth_enterprise/rpms/shared/lib/key-value.sh]: It would be a good idea to write some unit tests for the key/value store.

Comment (tst-lsavoie) [reply earth_enterprise/rpms/shared/lib/key-value.sh]: Did you write this code or is this copied from somewhere?  If it's from somewhere else, we'll need to attribute it and state the license.

Comment (tst-ppenev) [reply earth_enterprise/rpms/shared/lib/key-value.sh]: It's my code.

Comment (tst-ppenev) [reply earth_enterprise/rpms/shared/lib/key-value.sh]: Unit tests for the key-value store seems like a good sub-task we can give to someone.  Would you like to write it, or should I do that?

Comment (tst-lsavoie) [reply earth_enterprise/rpms/shared/lib/key-value.sh]: I can write the ticket.

File: earth_enterprise/rpms/build.gradle [hunk 3]
```diff
@@ -285,16 +287,45 @@ task openGeeServerDeb (type: Deb) {
...
-    requires('opengee-common', openGeeVersion, GREATER | EQUAL)
     packageName = 'opengee-fusion'
+    release = '1'
+    version = openGeeVersion
+    user = 'root'
+    permissionGroup = 'root'
+    packageGroup = 'Application/Productivity'
+    summary = 'Open GEE Fusion UI and Command-line Utilities'
+    packageDescription =
+'''Command line utitiles and Qt UI for building Open GEE globes.
```
Comment (tst-lsavoie) [inline earth_enterprise/rpms/build.gradle]: Minor: Is it "Qt" or "QT"?

**Pull request merged**